Formula/kube-aws.rb

```diff
@@ -0,0 +1,44 @@
+require "yaml"
+
+class KubeAws < Formula
+  desc "Kubernetes on AWS"
+  homepage "https://github.com/coreos/coreos-kubernetes/tree/master/multi-node/aws"
+  url "https://github.com/coreos/coreos-kubernetes/archive/v0.8.1.tar.gz"
+  sha256 "fe65c3fb72af7aa47dad07b8428eace077314edd3c129dba1fea5d3cccfaf094"
+  head "https://github.com/coreos/coreos-kubernetes.git"
+
+  depends_on "go" => :build
+
+  def install
+    executable = "bin/kube-aws"
+    ENV["GOPATH_VENDOR"] = "#{Dir.pwd}/_gopath-vendor"
+    ENV["GOPATH_KUBE_AWS"] = "#{Dir.pwd}/_gopath-kube-aws"
+    ENV["KUBE_AWS_DIR"] = "#{ENV["GOPATH_KUBE_AWS"]}/src/github.com/coreos/coreos-kubernetes/multi-node/aws"
+    ENV["GOPATH"] = "#{ENV["GOPATH_VENDOR"]}:#{ENV["GOPATH_KUBE_AWS"]}"
+
+    Dir.chdir "#{Dir.pwd}/multi-node/aws/"
+
+    rm_rf [ENV["GOPATH_VENDOR"], ENV["GOPATH_KUBE_AWS"]]
+
+    mkdir_p ENV["GOPATH_VENDOR"]
+    mkdir_p File.dirname(ENV["KUBE_AWS_DIR"])
+
+    ln "#{Dir.pwd}/vendor", "#{ENV["GOPATH_VENDOR"]}/src"
+    ln Dir.pwd.to_s, (ENV["KUBE_AWS_DIR"]).to_s
+
+    system "go", "generate", "./pkg/config"
+    system "go", "build", "-ldflags", "-X github.com/coreos/coreos-kubernetes/multi-node/aws/pkg/cluster.VERSION=#{version}", "-a", "-tags", "netgo", "-installsuffix", "netgo", "-o", executable.to_s, "./cmd/kube-aws"
```

Comment:
Replace `executable.to_s` with `bin/"kube-aws"` and you can delete the `bin.install` line below.  Then you can remove the `executable` variable entirely.

Also, please add a line break or two here for readability.